dealership/vehicles.py

```diff
@@ -1,15 +1,38 @@
+# working
+
+# improvements to be made:
+# make sale_price and purchase_price methods DRYer
+
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
-
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
+        
 class Car(Vehicle):
-    pass
+    def sale_price(self):
```

Comment:
The sale_price and purchase_price method can be defined in the Vehicle superclass leaving only the hardcoded sale and purchase multipliers as the attributes of the subclass. This may improve readability by avoiding duplicates of the methods within each subclasses